ocw-cli/cli_app.py

```diff
@@ -115,7 +119,7 @@ def load_local_model_screen(header):
                    note = "WARNING: Model file cannot be added."
               elif answer == "1":
                    model_dataset = load_file(model_path, variable_name)
-                   model_datasets.append(model_dataset)
+                   model_datasets.append(dsp.normalize_dataset_datetimes(model_dataset, model_dataset.temporal_resolution()))
```

Comment:
this part of code is just for adding and removing model dataset to the list of models, there should not be any data processing happening here. You should move the normalize processing down to the processing section.